submissions/akulahere/html-css-popup/css/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
```

Comment:
Try to avoid unnecessary comments.

Reply:
That's initial comments from normalize.css, not his

Reply:
I know it and if I correctly understand, he can download this file without comments.